### PR TITLE
Anne/views all

### PR DESCRIPTION
**Pull request by annesbln**
I added links for editing and deleting events in the overall events list. I added a link for creating a new event at the end. I didn't do buttons as this seemed to be much more complicated. Hope this is good enough for the beginning... Bare with me. This is my very first pull request :octocat:

### CODE REVIEW

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h1>All Events</h1>
+<table>
+  <% @events.each do |event| %>
+    <tr>
+      <td><%= event.title %></td>
+      <td><%= link_to("Edit", events_path(event)) %></td>
+      <td><%= link_to("Delete", events_path) %></td>
```
Comment (yakamozzz) [inline app/views/events/index.html.erb]: hi, muss vielleicht bei delete auch noch das argument (event) an den events_path? bzw, wenn nicht, warum nicht?

Comment (yakamozzz) [reply app/views/events/index.html.erb]: und es klappt auch nicht mit dem loeschen, oder? bei meinem anderen projekt sieht es so aus: <td><%= link_to 'Destroy', story_path(story),
              method: :delete,
              data: { confirm: 'Are you sure?' } %></td>
...hmm.

Comment (annesbln) [reply app/views/events/index.html.erb]: Thanks for your valuable thoughts on this. :100: I think the delete link should work with my newest commit. However, now there is a problem in events_controller showing that the event_params method does not work properly.

Comment (kolja-lucht) [reply app/views/events/index.html.erb]: Was fixed in 99e4dafbf35b125d7503f70fbbcdc8f2991c7e0a

**Pull request merged**